frontend/lib/verification/scanner/verification_encoder.dart

```diff
@@ -19,26 +18,9 @@ String encodeVerificationCardDetails(VerificationCardDetails verificationCardDet
         ),
       ),
     ),
-    hashSecret: const Base64Decoder().convert(cardDetails.hashSecretBase64),
+    pepper: cardDetails.pepper,
     otp: verificationCardDetails.otp,
   );
 
-  return const Base64Encoder().convert(verifyCode.writeToBuffer());
-}
-
-// TODO (Max): Refactor into Dart extension
-VerificationCardDetails decodeVerificationCardDetails(String base64Data) {
-  const base64Decoder = Base64Decoder();
-
-  final verifyCode = CardVerifyCode.fromBuffer(base64Decoder.convert(base64Data));
-  final cardInfo = verifyCode.info;
-
-  final fullName = cardInfo.fullName;
-  final randomBytes = const Base64Encoder().convert(verifyCode.hashSecret);
-  final expirationDay = cardInfo.expirationDay;
-  final cardType = CardType.values[cardInfo.extensions.extensionBavariaCardType.cardType.value];
-  final regionId = cardInfo.extensions.extensionRegion.regionId;
-  final otp = verifyCode.otp;
-
-  return VerificationCardDetails(BaseCardDetails(fullName, randomBytes, expirationDay, cardType, regionId), otp);
+  return const Base64Encoder().convert(QrCode(dynamicVerify: verifyCode).writeToBuffer());
 }
```

Comment:
Where did this go? Was this not used anywhere?

Reply:
Nope, dead code